### ISSUE TITLE
Unable to fully match, in fluent-bit.

### ISSUE THREAD

**Issue by yonchin** [fluent/fluent-bit#598]
I customize the audit log, but only part of it. Here is my regular expression http://rubular.com/r/cowa3peJlu:

`
^type=(?<audit>[^ ]*) msg=(?<msg>[^ ]*): pid=(?<pid>[^ ]*) uid=(?<uid>[^ ]*) auid=(?<auid>[^ ]*) ses=(?<ses>[^ ]*) msg=(?<message>'.*')$
`

This is the output log：

`
flbt    | [9294] audit: [1526901058.484943331, {"audit"=>"CRYPTO_KEY_USER", "msg"=>"audit(1526890234.635:11390564)", "pid"=>"15995", "uid"=>"0", "auid"=>"4294967295", "ses"=>"4294967295"}]
flbt    | [9295] audit: [1526901058.484943331, #<UNKNOWN 1702051172 3619268344645969440>]
flbt    | [9296] audit: [1526901058.484943331, #<UNKNOWN 1648193638 4123669127743355194>]
flbt    | [9297] audit: [1526901058.484947716, {"audit"=>"CRYPTO_KEY_USER", "msg"=>"audit(1526890234.635:11390565)", "pid"=>"15995", "uid"=>"0", "auid"=>"4294967295", "ses"=>"4294967295"}]
flbt    | [9298] audit: [1526901058.484947716, #<UNKNOWN 1702051172 3907496315615995424>]
flbt    | [9299] audit: [1526901058.484947716, #<UNKNOWN 842887270 3475150970380433722>]
flbt    | [9300] audit: [1526901058.484951626, {"audit"=>"USER_LOGIN", "msg"=>"audit(1526890234.635:11390566)", "pid"=>"15995", "uid"=>"0", "auid"=>"4294967295", "ses"=>"4294967295"}]
flbt    | [9301] audit: [1526901058.484951626, #<UNKNOWN 1869574690 8247064187943871864>]
flbt    | [9302] audit: [1526901058.484951626, #<UNKNOWN 790773093 7526486049819288179>]
`

Please give advice or comments，tks

**Comment by edsiper** [comment]
thanks for reporting this. I was able to reproduce the problem only if in my parser I defined a time key which don't exists in the record, e.g:

```
[PARSER] 
    Name   auditd
    Format regex
    Regex  /^type=(?<audit>[^ ]*) msg=(?<msg>[^ ]*): pid=(?<pid>[^ ]*) uid=(?<uid>[^ ]*) auid=(?<auid>[^ ]*) ses=(?<ses>[^ ]*) msg=(?<message>'.*')$/
    Time_Key time
    Time_Format %d/%b/%Y:%H:%M:%S %z
```

the workaround for now will be to remove the Time_Key and Time_Format properties from the parser. I will will arrive shortly.

**Comment by edsiper** [comment]
FYI: The issue has been fixed on GIT master (749e3f3b13b9ba82a71790acd138101eb0b18ec1) and will be available in the next release of v0.13.3 (shortly).

thanks again for your help.
